FIX-#3624: Reformat modin readthedocs theme to look closer to pandas. (#3726)
modin-project/modin: docs/contact.rst

Contact
=======

Slack
-----

> Join the `Modin Slack`_ to join the conversation around making Modin better,

Reviewer: ```suggestion
Join our `Slack`_ community to connect with Modin users and contributors,
```

> ask questions to the community, and meet other Modin users.

Reviewer: ```suggestion
discuss, and ask questions about all things Modin-related.
```

Mailing List
------------

General questions, potential contributors, and ideas can be directed to the
`developer mailing list`_. It is an open Google Group, so feel free to join anytime! If
you are unsure about where to ask or post something, the mailing list is a good place to
ask as well.

Issues
------

> Bug reports and feature requests can be directed to the issues_ page of the Modin

Author: Added a section linking to our Discourse forum

GitHub repo.

> .. _Modin slack: https://modin.org/slack.html

Reviewer: ```suggestion
.. _slack: https://modin.org/slack.html
```
